### PR TITLE
net-address: Added support for PSA ranges

### PR DESCRIPTION
**Pull request by sruffilli**

### CODE REVIEW

File: modules/net-address/README.md [hunk 1]
```diff
@@ -47,6 +47,23 @@ module "addresses" {
...
 
+### PSA addresses
+
+```hcl
+module "addresses" {
+  source     = "./modules/net-address"
+  project_id = var.project_id
+  psa_addresses = {
+    cloudsql-mysql = {
+      address       = "10.10.10.0"
```
Comment (ludoo) [inline modules/net-address/README.md:58]: Would it make sense to use `range` so as to avoid the need for an extra attribute?

**Review by ludoo — APPROVED** [review]
LGTM, nit on variable type

File: modules/net-address/README.md (hunk 1 above)
Comment (sruffilli) [reply modules/net-address/README.md:58]: Makes sense - the reasoning behind not doing that was to keep a the interface consistent with psc_addresses and the resource itself. It's a trivial change, your call.

Comment (ludoo) [reply modules/net-address/README.md:58]: I'm all for simplifying the interface (cf the GKE module for extreme examples of the approach). It's a minor thing, so it's also ok to leave it like this.

**Pull request merged**